doc/default.nix

```
@@ -99,7 +99,10 @@ in pkgs.stdenv.mkDerivation {
     nixos-render-docs
   ];
 
-  src = ./.;
+  src = pkgs.lib.fileset.toSource {
+    root = ./.;
+    fileset = pkgs.lib.fileset.difference ./. (pkgs.lib.fileset.unions [ ./common.nix ./default.nix ./shell.nix ]);
```

Side notes:

- I frequently find myself repeating some kind of a [`fileFilter`](https://github.com/SomeoneSerge/dust3r.nix/blob/2c5ed64144bc95ef504de21083a13c03092c40fb/package.nix#L37) to discard the nix files. Makes me wonder if we want a shorthand for this...
- Can unnest e.g. via [`lib.pipe`](https://github.com/SomeoneSerge/dust3r.nix/blob/2c5ed64144bc95ef504de21083a13c03092c40fb/package.nix#L35)

---

As mentioned earlier, I'd really recommend explicitly selecting the files to include:

```suggestion
    fileset = pkgs.lib.fileset.unions [ ... ];
```

That's going to be much more resilient.